research/eip-draft_token_reputation.md

```diff
@@ -37,6 +37,14 @@ These two EIPs also lack the emittance of a `block.number` within their events.
 
 For reference, [OpenSea](https://docs.opensea.io/docs/metadata-standards#disable-trading-for-staked-or-locked-tokens) leverages both of the events noted in EIP-5192 and EIP-5753 to disable trading for staked/locked tokens. There are other accepted events by OpenSea, including the `Stake` and `Unstake` events outlined in this proposal, which _do not_ have an EIP associated with them. Although it is not necessarily the primary motivation of this token reputation EIP, it's been noted to demonstrate how marketplaces are using this EIP's event definitions today. In other words, it is true that the events outlined in this proposal **will disable/enable trading a token on marketplaces**; this proposal _already_ has real-world usage.
 
+### Transferrable vs. Non-Transferrable
```

Comment:
Related to the below comment, I think it would be useful to show some examples of selecting reputation w/ SQL for different use cases: 
1. the rigs use case: non-transferrable (select on address)
2. game "points" or similar (build up a character and sell it or something): transferrable (select on token id)